CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 0.1.5 - in progress
 
+### Added
+- Initial slider/selection values and light theme for channel controller.
```

Comment:
The slider changes aren't a part of this PR, right?

Reply:
Right - I forgot to update the changelog and never got notified about it because there was a release off master while I was working on that.